### PR TITLE
Test against Plone 5.1.x

### PR DESCRIPTION
**Pull request by mbaechtold**
🚧 DO NOT MERGE, WAITING FOR `ftw.publisher.*` BEING COMPATIBLE WITH PLONE 5.1 🚧

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ is shown on the current context.
 Compatibility
 =============
 
-Supports Plone `4.2`, `4.3`.
+Supports Plone `4.2`, `4.3` and `5.1`.
```
Comment (jone) [inline README.rst]: `- 4.2` 😉

File: setup.py [hunk 2]
```diff
@@ -35,6 +36,7 @@ def read(*rnames):
         'Framework :: Plone',
         'Framework :: Plone :: 4.2',
```
Comment (jone) [inline setup.py]: ✂️

File: test-plone-5.1.x.cfg [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+extensions += mr.developer
+
+auto-checkout =
+    ftw.publisher.core
+    ftw.publisher.receiver
+
+[branches]
+ftw.publisher.core = ne/plone-5.1
+ftw.publisher.receiver = plone-5.1.x
```
Comment (jone) [inline test-plone-5.1.x.cfg]: 🚧 I'd wait for releases here and remove it before merging, so that we start with a clean state.

**Comment by djowett-ftw** [comment]
Superceeded by #34

**Pull request closed without merging**